deployment-templates/py/forseti-instance.py

```diff
@@ -118,8 +119,9 @@ def GenerateConfig(context):
 PROTOC_PATH=$(which protoc)
 if [ -z "$PROTOC_PATH" ]; then
         cd $USER_HOME
-        wget https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-linux-x86_64.zip
-        unzip -o protoc-3.2.0-linux-x86_64.zip
+        PROTOC_DOWNLOAD_URL=$(curl -s $FORSETI_PROTOC_URL)
```

Comment:
nit: It seems a bit odd that we are curling for the url.  Is there a way to just read it from the file locally?  Okay if you elect to keep this.